karyon3-api/src/main/java/com/netflix/karyon/annotations/Priority.java

```diff
@@ -0,0 +1,21 @@
+package com.netflix.karyon.annotations;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Use Priority with classes that are auto-loaded (such as AutoBinder) to control
+ * the order in which these auto-loaded classes are called.  
```

Comment:
What is the resolution method for multiple items at the same priority?  Random, asciibetical, etc?  Also, why a priority and not a graph of dependencies?  It seems to me that having a directed graph of dependencies will guarantee a proper ordering without user interaction?